kano_settings/system/advanced.py

```diff
@@ -342,32 +349,34 @@ def set_ultimate_parental(ultimate, safesearch):
         restore_dns_interfaces()
         redirect_traffic_to_google()
         if ultimate:
+            set_setting("use_sentry", "whitelist")
             parse_whitelist_to_config_file(sentry_config)
         elif safesearch:
```

Comment:
They are mutually exclusive (you can't access search engines in ultimate mode anyway).